app/src/components/connection_status/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Web3Consumer } from 'web3-react'
+
+const ConnectionStatus: React.FC = () => {
+  return (
+    <Web3Consumer>
+      {(context: any) => {
+        const { active, account, networkId } = context
+        return (
+          active && (
+            <React.Fragment>
```

Comment:
`<>`

Reply:
Updated to use the short syntax. Thanks for all the feedback